docker/hq-compose.yml

```diff
@@ -73,7 +73,7 @@ services:
       - ${VOLUME_PREFIX}${BLANK_IF_TESTS-redis:}/data
 
   elasticsearch:
-    image: elasticsearch:7.6.2
+    image: elasticsearch:7.9.1
```

Comment:
Will this change be a seamless upgrade for developer envs? Like is it a drop-in replacement that is fully backward compatible with existing data?

Reply:
Yes, all of the code is backwards compatible. The main reason behind the upgrade is that some features are replaced in 7.9 with new features, these old features continue to work in 7.9 but will be discontinued in v8. So, we might as well use new features. These are mainly required for ICDS but it will be good to be on the latest anyway.